### PR TITLE
Rc default for low and high

### PR DESCRIPTION
**Pull request by khancyr**
rework from https://github.com/ArduPilot/ardupilot/pull/8000

### CODE REVIEW

File: libraries/AP_Arming/AP_Arming.cpp [hunk 1]
```diff
@@ -1109,11 +1109,11 @@ bool AP_Arming::rc_checks_copter_sub(const bool display_failure, const RC_Channe
         const RC_Channel *channel = channels[i];
         const char *channel_name = channel_names[i];
         // check if radio has been calibrated
-        if (channel->get_radio_min() > 1300) {
```
Comment (peterbarker) [inline libraries/AP_Arming/AP_Arming.cpp:1112]: There are two distinct uses of this pair of numbers (1300 and 1700).

One is setting strict endpoints for calibration values.

The other are a poor-man's `RC_OPTION` switch threshold values.

They should not be conflated.

File: libraries/RC_Channel/RC_Channel.h [hunk 2]
```diff
@@ -99,6 +99,9 @@ class RC_Channel {
     void init_aux();
     bool read_aux();
 
+    static const uint16_t RC_LOW_DEFAULT = 1300;
```
Comment (peterbarker) [inline libraries/RC_Channel/RC_Channel.h:102]: Not variables.  It's right there in the declaration :-P

Also not sure what the "DEFAULT" is in aid of.

I'd also point out that where these are used we should really be using `AUX_PWM_TRIGGER_LOW` and `AUX_PWM_TRIGGER_HIGH`.  Sadly, these are different values to the constants here.  When we move those functions to be RC_OPTIONs we run the risk of breaking people's setups until they fix their transmitter ranges :-(

Comment (tridge) [inline libraries/RC_Channel/RC_Channel.h:102]: add a comment explaining what these are for

**Comment by peterbarker** [comment]
Please ensure this either retains existing code size of preferably makes it smaller :-)

**Comment by peterbarker** [comment]
@khancyr needs a rebase and some work based on reviews.

**Comment by peterbarker** [comment]
@khancyr needs a chunk of work now.  Still interested in this particular PR?

**Comment by khancyr** [comment]
@peterbarker done ! I think I address the review. I have check that it doesn't change the build size. So that is pretty NFC.

**Review by tridge — APPROVED** [review]

File: libraries/RC_Channel/RC_Channel.h [hunk 3]
```diff
@@ -240,11 +240,20 @@ class RC_Channel {
 #if !HAL_MINIMIZE_FEATURES
     const char *string_for_aux_function(AUX_FUNC function) const;
 #endif
+    // pwm value above which we condider that Radio min value is invalid
```
Comment (amilcarlucas) [inline libraries/RC_Channel/RC_Channel.h:243]: s/condider/consider/

File: libraries/RC_Channel/RC_Channel.h [hunk 4]
```diff
@@ -240,11 +240,20 @@ class RC_Channel {
...
+    static const uint16_t RC_CALIB_MIN_LIMIT_PWM = 1300;
+    // pwm value under which we condider that Radio max value is invalid
```
Comment (amilcarlucas) [inline libraries/RC_Channel/RC_Channel.h:245]: s/condider/consider/

**Review by amilcarlucas — COMMENTED** [review]
typos

**Pull request merged**